apps/tenant-management-webapp/src/app/components/AsideLinks.tsx

```diff
@@ -14,12 +14,12 @@ function AsideLinks({ serviceLink, docsLink }: AsideLinksProps): JSX.Element {
       <GapAdjustment>Helpful links</GapAdjustment>
       {serviceLink && (
         <HyperLinkColor>
-          <ExternalLink link={serviceLink} text="See the code" />
+          <ExternalLink testId="code-link" link={serviceLink} text="See the code" />
```

Comment:
Should this be data-testid?

Reply:
The property added to the external link is testId

Reply:
I just checked the code and it looks the ExternalLink has the "testId" property.